README.md

```diff
@@ -45,15 +45,16 @@ struct Counter: View {
   let limit: Int
 
   var body: some View {
```

Comment:
I think you need `@ViewBuilder` here

Reply:
Yes, I forgot to commit it 🤦‍♂️

Reply:
No longer needed as `@ViewBuilder` is already on the `body` declaration of the `View` protocol.